cmd/edenStart.go

```diff
@@ -22,6 +22,7 @@ var (
 	eserverLogFile   string
 	evePidFile       string
 	eveLogFile       string
+	eveRemote        bool
```

Comment:
Ok, after our discussion on Mon and after thinking about it some more -- I think I'd like to offer the following compromise for now: since it appears that eveRemote is only really useful for figuring out the routable IP address for the remote instance of EVE -- I suggest we introduce eve.remote.addr (or eve.remote_addr) instead.


Note that a big difference with the current implementation is that I'm suggesting we completely cut Adam out of the loop when it comes to figuring out EVE's routable IP address. Whatever EVE reports to Adam has a high chance of NOT being routable -- so there's not really much use to pay attention to it.

Having an ability to always explicitly set remote EVE's address (either via DNS name or IP) will be enough for now.

Of course, having a reasonable default for this new variable (like localhost or 127.0.0.1) will make all the sense.

@giggsoff @sadov please ping me on Slack if you need to discuss it further

Reply:
For now eve.remote also say us to start qemu or not.
Also, maybe without new eve.remote_addr (empty value in config) we will keep the current logic of IP getting from info? For rpi it works fine.

Reply:
so, I add eve.remote-addr option with default value 127.0.0.1